### ISSUE TITLE
flutter_simple_calculator >=2.0.0 depends on expressions ^0.2.3 which depends on petitparser ^4.0.2, 

### ISSUE THREAD

**Issue by sgehrman** [appsup-dart/expressions#22]
flutter_simple_calculator >=2.0.0 depends on expressions ^0.2.3 which depends on petitparser ^4.0.2,

**Comment by alexmarkley** [comment]
I opened a pull request https://github.com/appsup-dart/expressions/pull/23 to solve this issue.

In my case, the version solving error looks more like:

```
Because expressions >=0.2.0 depends on petitparser ^4.0.2 and xml >=5.4.0 depends on petitparser ^5.0.0, expressions >=0.2.0 is incompatible with xml >=5.4.0.
And because flutter_svg >=1.1.0 depends on xml ^6.0.1, expressions >=0.2.0 is incompatible with flutter_svg >=1.1.0.
So, because someapp depends on both flutter_svg ^1.1.0 and expressions ^0.2.3, version solving failed.
```

But I think this is the same issue.

**Comment by alexmarkley** [comment]
Hopefully a fix gets merged soon, but in the meantime I am using the following workaround in my pubspec.yaml:

```
dependencies:
  expressions:
    git:
      url: https://github.com/alexmarkley/expressions.git
      ref: petitparser5
```

**Comment by rbellens** [comment]
fixed since version 0.2.4
